Dockerfile

```diff
@@ -54,7 +54,15 @@ ENV \
     PHP_DISPLAY_ERRORS="1" \
     PHP_MEMORY_LIMIT="-1" \
     XHGUI_MONGO_URI="global-xhgui:27017" \
-    XHGUI_PROFILING="enabled"
+    XHGUI_PROFILING="enabled" \
+    TZ=Europe/Berlin \
```

Comment:
```suggestion
    TIMEZONE=Europe/Berlin \
```

Reply:
no, TZ is a docker recognized variable

Reply:
https://medium.com/developer-space/be-careful-while-playing-docker-about-timezone-configuration-e7a2217e9b76